client/components/Cart.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Icon } from 'react-materialize';
+import { connect } from 'react-redux';
+import { removeItem } from '../store/cart';
+
+function Cart(props) {
+  return (
+    <div className="container">
+      <div className="row center">
+        <h3>Cart</h3>
+      </div>
+      <div className="row">
+        <table className="highlight">
+          <thead>
+            <tr>
+              <th>Enhancement</th>
+              <th>Pet</th>
+              <th>Price</th>
+              <th>Quantity</th>
+              <th>Item Total</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            {
+              props.cart.map((item) => {
+                const itemEnhancement = props.enhancements.find(enhancement =>
+                  enhancement.id === item.enhancementId);
+                const itemAnimal = props.animals.find(animal => animal.id === item.animalId);
+                return (
+                  <tr key={item.id}>
+                    <td>{itemEnhancement.name}</td>
+                    <td>{itemAnimal.name}</td>
+                    <td>{+itemEnhancement.price + +itemAnimal.price}</td>
+                    <td>{item.quantity}</td>
+                    <td>{item.price}</td>
+                    <td><Button
+                      onClick={() => props.onDelete(item.id)}
+                      floating
+                      className="red"
+                      waves="light"
+                      icon="delete"
+                    /></td>
+                  </tr>
+                );
+              })
+            }
+          </tbody>
+        </table>
+      </div>
+      <div className="row">
+        <div className="col s6"><h3>Total: ${
+          props.cart.reduce((sum, item) => sum + +item.price, 0)
```

Comment:
I'm seeing quite a few examples like this where a value is calculated inline in the `<html>` part of the component.

It would be clearer and more intention revealing to name these values.

This could be done with a set of utility functions:

```js
import { sumCartTotal } from '../cartUtilities';
<h3>Total ${ sumCartTotal(props.cart) }</h3>
```

That might be overdoing it to start, a simple way to do this is to pull the values the component will use at the top of the render function:

```js
render () {
  const cartTotal = props.cart.reduce((sum, item) => sum + +item.price, 0);
  return <h3>Total: ${cartTotal}</h3>
}
```